tests_e2e/templates/execute-tests.yml

```diff
@@ -33,4 +33,22 @@ jobs:
           # Add all KeyVault secrets explicitly as they're not added by default to the environment vars
           AZURE_CLIENT_ID: $(AZURE-CLIENT-ID)
           AZURE_CLIENT_SECRET: $(AZURE-CLIENT-SECRET)
-          AZURE_TENANT_ID: $(AZURE-TENANT-ID)
\ No newline at end of file
+          AZURE_TENANT_ID: $(AZURE-TENANT-ID)
+
+      - task: CopyFiles@2
```

Comment:
created task to copy junit file to our harvest path.

I couldn't add as part of lisa run since it creates juit file after completion of test suite.

2022-11-08 22:46:34.705[139705718830912][INFO] lisa.notifier[JUnit] JUnit: /home/vsts/tmp/20221108/20221108-223952-777/lisa.junit.xml